tests/test_status.py

```diff
@@ -18,3 +19,35 @@ def test_service_status(self):
     def test_ui_status(self):
         res = requests.get(UI_URL)
         res.raise_for_status()
+
+
+class TestRedirect(unittest.TestCase):
+
+    def redirect(self, lang):
+        res = requests.get(
+            UI_URL, headers={'Accept-Language': lang}, allow_redirects=False)
+        assert res.status_code == 302
+        return urlparse.urlparse(res.headers['Location']).path
+
+    def test_dbg(self):
+        assert '/dbg/index.html' == self.redirect('dbg')
+
+    def test_en(self):
+        assert '/en/index.html' == self.redirect('en')
+
+    def test_dbg_en(self):
+        assert '/dbg/index.html' == self.redirect('dbg;q=0.7,en;q=0.3')
+
+    def test_fr(self):
```

Comment:
This test name confused me at first. A name like `test_unsupported_locale` would make it more clear (or you could add a comment).